contrib/completion/bash/docker

```diff
@@ -3670,6 +3672,8 @@ _docker_service_update() {
 # and `docker service update`
 _docker_service_update_and_create() {
 	local options_with_args="
+		--cap-add
+		--cap-drop
```

Comment:
This enables completion of `--cap-add` and `--cap-drop`.
To add completion for the specific capabilities, you need to add [this](https://github.com/docker/cli/blob/master/contrib/completion/bash/docker#L2020-L2027) to the case statement that starts [here].(https://github.com/docker/cli/blob/10b529e2aed166263f2ae5d7d9e38524b336f6e3/contrib/completion/bash/docker#L3834)

Reply:
Oh, thanks! Yes, missed that: I'll add those lines